spec/migrations/20160829134746_remove_atomic_container_providers_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_migration
+
+describe RemoveAtomicContainerProviders do
+  let(:ems_stub)  { migration_stub(:ExtManagementSystem) }
+  let(:auth_stub) { migration_stub(:Authentications) }
+  let(:workers_stub) { migration_stub(:MiqWorkers) }
+
+  migration_context :up do
+    it "Updates Atomic Provider to Openshift and Atomic Enterprise to Openshift Enterprise" do
+      e1 = ems_stub.create!(:type => "ManageIQ::Providers::Atomic::ContainerManager")
+      e2 = ems_stub.create!(:type => "ManageIQ::Providers::AtomicEnterprise::ContainerManager")
+      e3 = ems_stub.create!(:type => "EmsOther")
+      a1 = auth_stub.create!(:name => "ManageIQ::Providers::Atomic::ContainerManager Server1")
+      a2 = auth_stub.create!(:name => "ManageIQ::Providers::AtomicEnterprise::ContainerManager Server2")
+      a3 = auth_stub.create!(:name => "EmsOther Server3")
+      w1 = workers_stub.create!(:type => "ManageIQ::Providers::Atomic::ContainerManager::EventCatcher")
+      w2 = workers_stub.create!(:type => "ManageIQ::Providers::Atomic::ContainerManager::MetricsCollectorWorker")
+      w3 = workers_stub.create!(:type => "ManageIQ::Providers::Atomic::ContainerManager::RefreshWorker")
+      w4 = workers_stub.create!(:type => "ManageIQ::Providers::AtomicEnterprise::ContainerManager::EventCatcher")
+      w5 = workers_stub.create!(:type => "ManageIQ::Providers::AtomicEnterprise::ContainerManager::MetricsCollectorWorker")
+      w6 = workers_stub.create!(:type => "ManageIQ::Providers::AtomicEnterprise::ContainerManager::RefreshWorker")
+      w7 = workers_stub.create!(:type => "OtherWorker")
+
+      migrate
+      e1.reload
+      e2.reload
+      e3.reload
+      a1.reload
+      a2.reload
+      a3.reload
+      w1.reload
+      w2.reload
+      w3.reload
+      w4.reload
+      w5.reload
+      w6.reload
+      w7.reload
```

Comment:
@zgalor can't you use something like a dictionary? You would be able to automate the `reload` part.

```ruby
entities = {}
entities['ems1'] = ems_stub.create!(:type => "ManageIQ::Providers::Atomic::ContainerManager")
...

entities.each { |_, e| e.reload }

expect(entities['ems1'].type).to eq("ManageIQ::Providers::Openshift::ContainerManager")
...
```